app/templates/client/Gruntfile.js

```diff
@@ -421,13 +426,34 @@ module.exports = function (grunt) {
     buildcontrol: {
       dist: {
         options: {
-          remote: 'https://github.com/repoName.git',
+          remote: 'https://github.com/.git',
           branch: 'gh-pages',
           commit: true,
           push: true
         }
       }
+    },
+
+    // Prompt to ask productioon configuration.
+    prompt: {
```

Comment:
I prefer not to have prompts on the build process, so we can later use it in travis if we want.